Packs/Expanse/Integrations/Expanse/Expanse.yml

```diff
@@ -387,6 +391,201 @@ script:
     - contextPath: Expanse.Domain.WHOIS.Admin.Phone
       description: The phone number of the domain administrator
       type: String
+  - arguments:
+    - default: true
+      description: domain to search
+      isArray: true
+      name: common_name
+      required: true
+      secret: false
+    deprecated: false
+    description: certificate command
+    execution: false
+    name: certificate
+    outputs:
+    - contextPath: Certificate.SearchTerm
```

Comment:
delete the `Certificate.*` outputs/context paths. Leave just the `Expanse.Certificate.*`
We don't have Certificate standard reputation. All our standard outputs https://xsoar.pan.dev/docs/integrations/context-standards